packages/core/src/forms/control-action/control-action.element.ts

```diff
@@ -56,8 +76,9 @@ export class CdsControlAction extends CdsBaseButton {
       this.setSlotLocation();
     }
 
-    if (props.has('readonly')) {
+    if (props.has('readonly') || props.has('ariaLabel')) {
```

Comment:
Allows for readonly control actions to still be read. Also allows for readonly control actions to be skipped when reading them would be redundant.